src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -2515,6 +2515,18 @@ bool MessageHelper::SendUnsubscribedWayPoints(ApplicationManager& app_mngr) {
   return app_mngr.GetRPCService().ManageHMICommand(result);
 }
 
+bool MessageHelper::SendSubscribedWayPoints(ApplicationManager& app_mngr) {
```

Comment:
@ychernysheva just a suggestion. This method seems to be very similar to previous two methods. How about making a single method with a switch, for example?

Reply:
@mkorniichuk Good point, please look at requested changes in 0a5ceb735f31f840161aacb7a9231a49c5d192e4